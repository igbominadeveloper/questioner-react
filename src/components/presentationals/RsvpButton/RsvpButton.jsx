import React from 'react';

const RsvpButton = ({ onClick, className, text }) => (
  <p className="p-1 border br-20 mr-5" onClick={onClick}>
    <a href="#" className={`text-heavy p-10 br-20 ${className}`}>
      {text}
    </a>
  </p>
);

export default RsvpButton;
