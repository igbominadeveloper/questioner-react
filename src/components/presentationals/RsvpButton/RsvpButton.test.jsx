import React from 'react';
import { shallow } from 'enzyme';

import RsvpButton from './RsvpButton';

const props = {
  onClick: jest.fn(),
  className: '',
  text: '',
};

describe('RsvpButton', () => {
  describe('Render', () => {
    it('renders without error', () => {
      const component = shallow(<RsvpButton {...props} />);
      expect(component).toMatchSnapshot();
    });
  });
});
