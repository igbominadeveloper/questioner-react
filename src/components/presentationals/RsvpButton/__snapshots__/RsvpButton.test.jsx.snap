// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RsvpButton Render renders without error 1`] = `
<p
  className="p-1 border br-20 mr-5"
  onClick={[MockFunction]}
>
  <a
    className="text-heavy p-10 br-20 "
    href="#"
  />
</p>
`;
